server/service/src/sync/test/pull_and_push.rs

```diff
@@ -120,17 +120,18 @@ async fn test_sync_pull_and_push() {
         .map(|r| (r.record_id.clone(), r.table_name.clone()))
         .collect::<Vec<(String, String)>>();
 
-    let mut i = 0;
-    for translated_record in changelog_translated_records_id_and_table_name.clone() {
+    for (i, translated_record) in changelog_translated_records_id_and_table_name
```

Comment:
Side quest... just a tiny refactor... enumerate already does what we want, so no need for a mut i